VoodooRMI/Transports/I2C/RMII2C.cpp

```diff
@@ -313,6 +313,13 @@ int RMII2C::readBlock(u16 rmiaddr, u8 *databuff, size_t len) {
     if (i2cInput[2] != RMI_READ_DATA_REPORT_ID) {
         IOLogError("%s::%s RMI_READ_DATA_REPORT_ID mismatch %d", getName(), name, i2cInput[2]);
         retval = -1;
+        char *buf = new char[len*2 + 9];
+        for (int i=0; i<len+4; i++)
+            snprintf(buf + 2*i, 3, "%2d", i2cInput[i]);
```

Comment:
probably want `x` here rather than `d`, especially since with d, it could go up to three digits.

Reply:
lol, I grabbed it from a stash and forgot to check it.